src/pages/education.tsx

```diff
@@ -0,0 +1,218 @@
+                            />
+                            </svg>
+                        </div>
+                    </Hero>
+                </Container>
+                <Container>
+                    <SectionFeatures>
+                        <h3>How to enhance the learning process with Gitpod</h3>
+                            <div className="features">
+                                <div className="feature">
+                                    <h4>No setup</h4>
+                                    <p>Don't waste time on tedious setup. With Gitpod developer enviroments are automated, reproducible and versioned.</p>
+                                </div>
+                                <div className="feature">
+                                    <h4>Works on every device</h4>
+                                    <p>With Gitpod coding gets accessible for  everyone your students can contribute with a Chromebook or a tablet.</p>
+                                </div>
+                                <div className="feature">
+                                    <h4>Enhances the workflow</h4>
+                                    <p>Create customized lessons for your students. Everyone can start working on new asignments within minutes.</p>
+                                </div>
+                            </div>
+                            <div className="features">
+                                <div className="feature">
+                                    <h4>Collaborative</h4>
+                                    <p>Gitpod strengthens collaboration between your students. Code reviews has never been this easy and sharable snapshots foster the team workflow.</p>
+                                </div>
+                                <div className="feature">
+                                    <h4>Continous Support</h4>
+                                    <p>Easy maintenance, flexible subscriptions and continous support if needed.</p>
+                                </div>
+                                <div className="feature">
+                                    <h4>High Security</h4>
+                                    <p>Data integrity is of utmost importance. Gitpod has high security standards to keep your students and their data safe.</p>
+                                </div>
+                            </div>
+                    </SectionFeatures>
+                </Container>
+                <HeadingMission>Our Mission is to convey our passion for coding to students by eliminating barriers.</HeadingMission>
+                <SectionPricing>
+                    <Container>
+                        <h3>Pricing</h3>
+                        <p>We Love to support educational programs. That's why we have a specail pricing for universities, colleges, schools and non-comercail courses. Students also have the opritunity to get and individual subscription for 75% less than usual.</p>
```

Comment:
Typos + a suggestion:

```suggestion
                        <p>We love to support educational programs. That's why we have a special pricing for universities, colleges, schools and non-commercial courses. Students also have the opportunity to get an individual subscription for 75% off.</p>
```